src/main/java/io/github/redouane59/twitter/dto/tweet/TweetV2.java

```diff
@@ -313,6 +321,12 @@ public TweetType getTweetType() {
       return getReferencedTweets().get(0).getType();
     }
 
+    @Override
+    public List<MediaEntityV2> getMedia() {
+      LOGGER.error(NOT_IMPLEMENTED_EXCEPTION);
+      return null;
```

Comment:
isn't it better to return an empty list ?

Reply:
I agree, that we can do a `Collections.emptyList()` here.

I am asking myself the question, if a real exception would be the better way to go here, i.e. `UnsupportedOperationException` + override the JavaDoc. Would enable to fail early and notifies the user, that he/she shouldn't use this method for a V1 or V2 entity. Maybe in an enhancement PR, wdyt?